src/main/java/com/aws/greengrass/deployment/DeploymentService.java

```diff
@@ -228,8 +228,7 @@ protected void startup() throws InterruptedException {
                     // Handle IoT Jobs cancellation
 
                     if (currentDeploymentTaskMetadata != null && currentDeploymentTaskMetadata.getDeploymentType()
-                            .equals(nextDeployment.getDeploymentType())
-                            && currentDeploymentTaskMetadata.isCancellable()) {
+                            .equals(nextDeployment.getDeploymentType())) {
```

Comment:
We can still keep this check.

Reply:
I see that we're removing `cancellable` field altogether.

Reply:
The boolean field for this was removed because both Default Deployment and Kernel Update Deployment are cancellable now, it will always evaluate to true